ckanext/datastore/logic/action.py

```diff
@@ -158,6 +158,31 @@ def datastore_create(context, data_dict):
     return result
 
 
+def datastore_trigger_each_row(context, data_dict):
+    ''' update each record with trigger
+
+    The datastore_trigger_each_row API action allows you to apply triggers to
+    an existing DataStore resource.
+
+    :param resource_id: resource id that the data is going to be stored under.
+    :type resource_id: string
+
+    **Results:**
+
+    :returns: The rowcount in the table.
+    :rtype: int
+
+    '''
+    res_id = data_dict['resource_id']
+    p.toolkit.check_access('datastore_trigger_each_row', context, data_dict)
+
+    connection = db.get_write_engine().connect()
+
+    sql = sqlalchemy.text(u'''update "{0}" set _id=_id '''.format(res_id))
+    results = connection.execute(sql)
```

Comment:
this is what I mean https://github.com/ckan/ckan/blob/master/ckanext/datastore/db.py#L788-L791